Dockerfile.dev

```diff
@@ -36,8 +36,7 @@ COPY apps/fz_vpn/mix.exs /var/app/apps/fz_vpn/mix.exs
 COPY apps/fz_wall/mix.exs /var/app/apps/fz_wall/mix.exs
 COPY mix.exs /var/app/mix.exs
 COPY mix.lock /var/app/mix.lock
-RUN mix do deps.get --only dev, deps.compile --only dev, compile --only dev
-RUN mix do deps.get --only test, deps.compile --only test, compile --only test
+RUN mix do deps.get, deps.compile, compile
```

Comment:
We want to use `--only prod` for containers, there is no point to package and test/dev deps, at least for production one.

Reply:
This is for the dev / test env -- though most of the time I do run tests bare metal.

I think we standardized on Docker as a team to make it easier to test some of the eBPF stuff (Docker for Mac runs Linux 5.10) but it's not without its downsides...